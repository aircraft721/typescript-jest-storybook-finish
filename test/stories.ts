
import * as path from 'path';
import initStoryshots, { imageSnapshot } from '@storybook/addon-storyshots';

const pathToStorybookStatic = path.join(__dirname, '../', 'storybook-static');

initStoryshots({
  suite: 'Image snapshots',
  storyKindRegex: /Wakka$/,
  framework: 'react',
  test: imageSnapshot({
    storybookUrl: `file://${pathToStorybookStatic}`,
    getMatchOptions: () => ({
      failureThreshold: 0.02, // 2% threshold,
      failureThresholdType: 'percent',
    }),
  }),
});
