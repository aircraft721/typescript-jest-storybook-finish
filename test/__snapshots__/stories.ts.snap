// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots TypeScript and Storybook Sample Widget 1`] = `
<div>
  <h1>
    Hello 
    Michael
  </h1>
</div>
`;
